### DISCUSSION TITLE
Sponsor/Donate options?

### DISCUSSION THREAD

**Discussion by krayon** [picocomputer/community#130] [Q&A]
Hi @rumbledethumps ,

Just stumbled onto your work. The design, detail, polish and your sense of humour are simply superb.

My problems are that:
  1. Your YouTube videos don't link to Patreon;
  2. When I found you on Patreon, sponsorship doesn't appear to be an option (at least for me the option wasn't there); and
  3. You don't accept GitHub Sponsorship.

So how are we meant to sponsor your amazing efforts?! :grinning:

I'd ask you to please accept GitHub Sponsorship but I'm obviously biased :wink:

**Comment by rumbledethumps** [comment]
The time for financial support has passed. Enjoy the project.
